l10n-ms/src/main/java/com/example/l10nadmin/L10nApplication.java

```diff
@@ -17,4 +21,13 @@ public static void main(String[] args) {
 		SpringApplication.run(L10nApplication.class, args);
 	}
 
+	@Bean
+	public WebMvcConfigurer corsConfigurer() {
+		return new WebMvcConfigurerAdapter() {
```

Comment:
```suggestion
		return new WebMvcConfigurer() {
```
Since Spring 5+ the interface can be used directly, no Adapter needed. See https://www.baeldung.com/web-mvc-configurer-adapter-deprecated.

Reply:
Done!!